src/DynamoCoreWpf/ViewModels/Search/SearchCategory.cs

```diff
@@ -1,19 +1,38 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
 using Dynamo.Search.SearchElements;
 using Dynamo.UI;
 using Dynamo.Wpf.ViewModels;
+using Dynamo.Core;
+using System.Windows.Input;
+using Dynamo.UI.Commands;
 
 namespace Dynamo.Search
 {
-    public class SearchCategory
+    public class SearchCategory : NotificationObject
     {
         private readonly ObservableCollection<NodeCategoryViewModel> classes;
         private readonly List<SearchMemberGroup> memberGroups;
 
         public string Name { get; private set; }
 
+        private bool isExpanded;
+        public bool IsExpanded
+        {
+            get
+            { 
+                return isExpanded;
+            }
+            set
+            {
+                if (value.Equals(isExpanded)) return;
+                isExpanded = value;
+                RaisePropertyChanged("IsExpanded");
+            }
+        }
+
```

Comment:
While this looks good, I am not really into having a public setter if it is not absolutely required. Can I suggest this:

```csharp
private bool isExpanded;
public bool IsExpanded { get { return isExpanded; } }

private void OnClicked(object obj)
{
    isExpanded = !isExpanded;
    RaisePropertyChanged("IsExpanded");
}
```